integrationtest/engine-performance/src/main/java/org/hibernate/search/engineperformance/JMHBenchmarks.java

```diff
@@ -33,4 +42,31 @@ public void simpleIndexing(EngineHolder eh) {
 		tc.end();
 	}
 
+	@Benchmark
+	@Threads(20)
+	public void queryBooksByBestRating(QueryEngineHolder eh) {
+		SearchIntegrator searchIntegrator = eh.si;
+		Query luceneQuery = searchIntegrator.buildQueryBuilder()
+				.forEntity( BookEntity.class )
+				.get()
+				.all()
+				.createQuery();
+
+		long expectedIndexSize = eh.getExpectedIndexSize();
+		int maxResults = eh.getMaxResults();
+
+		HSQuery hsQuery = searchIntegrator.createHSQuery().luceneQuery( luceneQuery );
+		hsQuery.targetedEntities( Arrays.<Class<?>>asList( BookEntity.class ) );
```

Comment:
it wasn't on purpose, just that this code is actually quite old :) I'll fix that.